src/components/Publish/Pricing/Fees.tsx

```diff
@@ -1,18 +1,27 @@
-import React, { ReactElement } from 'react'
+import React, { ReactElement, useEffect, useState } from 'react'
 import Tooltip from '@shared/atoms/Tooltip'
 import styles from './Fees.module.css'
 import { useField } from 'formik'
 import Input from '@shared/FormInput'
 import Error from './Error'
+import {
+  publisherMarketPoolSwapFee,
+  publisherMarketFixedSwapFee
+} from '../../../../app.config'
```

Comment:
within components we can use `const { appConfig } = useSiteMetadata()` but I guess in end doesn't really matter